src/providers/VoiceFocusProvider/index.tsx

```diff
@@ -32,18 +34,25 @@ interface VoiceFocusState {
 
 const VoiceFocusContext = createContext<VoiceFocusState | null>(null);
 
-const VoiceFocusProvider: React.FC<Props> = ({ spec, options, children }) => {
+const VoiceFocusProvider: React.FC<Props> = ({ 
+  spec, 
+  options, 
+  createMeetingResponse,
+  children 
+}) => {
   const [isVoiceFocusSupported, setIsVoiceFocusSupported] = useState<
     boolean | undefined
   >(undefined);
   const [voiceFocusDevice, setVoiceFocusDevice] =
     useState<VoiceFocusTransformDevice | null>(null);
   const [voiceFocusTransformer, setVoiceFocusTransformer] =
     useState<VoiceFocusDeviceTransformer | null>(null);
+    const [initialized, setInitialized] =
+    useState<Boolean>(false);
 
   // Make sure that minor changes to the spec don't result in recomputation:
   // Any value of `{}` and undefined are all considered the same.
-  const vfSpec = useMemoCompare(
+  let vfSpec = useMemoCompare(
```

Comment:
Why do we need to change it to `let` instead of `const`?

Reply:
change to const